syft/workers/base.py

```diff
@@ -380,7 +386,7 @@ def send(
             obj.id_at_origin = obj.id
 
         # Send the object
-        self.send_obj(obj, worker)
+        self.send_obj(obj, worker, placeholder_id)
```

Comment:
This seems a bit awkward and reinforces my sense that it might be cleaner to package the `placeholder_id` into the `obj` somehow.